simulators/nav/src/components/NavSimulator.vue

```diff
@@ -359,13 +391,14 @@ export default class NavSimulator extends Vue {
 
       /* Subscriptions */
       [
-        { topic: '/autonomous',   type: 'Joystick' },
-        { topic: '/nav_status',    type: 'NavStatus' },
-        { topic: '/obstacle',     type: 'Obstacle' },
-        { topic: '/odometry',     type: 'Odometry' },
-        { topic: '/rr_drop_init', type: 'RepeaterDropInit' },
-        { topic: '/target_list',   type: 'TargetList' },
-        { topic: '/debugMessage', type: 'DebugMessage' }
+        { topic: '/autonomous',         type: 'Joystick' },
+        { topic: '/nav_status',         type: 'NavStatus' },
+        { topic: '/obstacle',           type: 'Obstacle' },
+        { topic: '/odometry',           type: 'Odometry' },
+        { topic: '/rr_drop_init',       type: 'RepeaterDropInit' },
+        { topic: '/target_list',        type: 'TargetList' },
+        { topic: '/zed_gimbal_command', type: 'ZedGimbalPosition' },
```

Comment:
should this not be  `topic: 'zed_gimbal_command', type: 'ZedGimbalCommand' `???
also fix the naming convention here (command --> cmd)

Reply:
That's not necessary. It wouldn't make sense to create a second LCM struct when one isn't needed. Just duplicated code that makes it harder to maintain.

Reply:
? so is the plan to receive and send the same struct?

Reply:
The same struct, yes. The same instance of that struct, no.

Reply:
okay this doesn't match the normal scheme or the ICD so that was confusing